src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md

```diff
@@ -343,6 +343,11 @@ Disconnect Sample](https://googlechrome.github.io/samples/web-bluetooth/device-d
 and the [Automatic Reconnect Sample](https://googlechrome.github.io/samples/web-bluetooth/automatic-reconnect.html)
 to dive deeper.
 
+Warning: Bluetooth GATT Services and Characteristics are invalidated when
+device gets disconnected. This means your code should always handle device
```

Comment:
Bluetooth GATT Attributes, services, characteristics, etc. are invalidated when a device disconnects. This means your code should always retrieve (through getPrimaryService(s), getCharacteristic(s), etc.) these attributes after reconnecting.